cdap-api/src/main/java/co/cask/cdap/api/dataset/lib/TimePartitionedFileSetArguments.java

```diff
@@ -54,6 +55,29 @@ public static Long getOutputPartitionTime(Map<String, String> arguments) {
   }
 
   /**
+   * This is the file extension for each partition when using TimePartitionedFileSet as an OutputFormatProvider.
+   * It's used to generate the end of the output file path for each partition.
+   * @param pathFormat The format for the path; for example: '%tF/%tH-%tM.%d',
```

Comment:
It doesn't tell what are all the available formats. E.g. what is `%tF`? Is it just the format string providing to `SimpleDateFormat`? If that the case, need to state it clear and have a `{@link SimpleDateFormat}`.